docs/interfaces/commands-and-shells.rst

```diff
@@ -48,127 +45,13 @@ UUID, run
-| ``det experiment list``                   |                                                    |
-+-------------------------------------------+----------------------------------------------------+
-| ``det -m 1.2.3.4 e`` |br|                 | Show information about experiments in the cluster  |
-| ``DET_MASTER=1.2.3.4 det e``              | at network address ``1.2.3.4``.                    |
-+-------------------------------------------+----------------------------------------------------+
-| ``det t logs -f 289``                     | Show the logs for trial 289 and continue showing   |
-|                                           | new logs as they arrive.                           |
-+-------------------------------------------+----------------------------------------------------+
-| ``det e label add 17 foobar``             | Add the label ``foobar`` to experiment 17.         |
-+-------------------------------------------+----------------------------------------------------+
-| ``det e describe 493 --metrics --csv``    | Display information about experiment 493,          |
-|                                           | including full metrics, in CSV format.             |
-+-------------------------------------------+----------------------------------------------------+
-| ``det e create -f --paused const.yaml .`` | Create an experiment with the configuration file   |
-|                                           | ``const.yaml`` and the code contained in the       |
-|                                           | current directory. The experiment is created in a  |
-|                                           | paused state, which means that it is not scheduled |
-|                                           | on the cluster until it is activated.              |
-+-------------------------------------------+----------------------------------------------------+
-| ``det e set max-slots 85 4``              | Ensure that experiment 85 does not use more than 4 |
-|                                           | slots in the cluster.                              |
-+-------------------------------------------+----------------------------------------------------+
-| ``det u create --admin hoid``             | Create a new user named ``hoid`` who has admin     |
-|                                           | privileges.                                        |
-+-------------------------------------------+----------------------------------------------------+
-| ``det version``                           | Show detailed information about the CLI and        |
-|                                           | master. This command does not take both an object  |
-|                                           | and an action.                                     |
-+-------------------------------------------+----------------------------------------------------+
-
 .. |br| raw:: html
 
    <br />
 
-********
- Shells
-********
+****************
+ Shell Commands
```

Comment:
I think it's clearer to leave this subheading as `Shells`, especially since above we've already described what `Commands` are. `Shell Commands` muddies the waters, to me.

Reply:
resolved